### ISSUE TITLE
As a developer, I want to investigate costs of changing warehouse size, size/frequency incoming trucks, linear/dependency on factors (EST 1)

### ISSUE THREAD

**Issue by JanVeen** [LCfP/Serious-Game-for-Kids#45]

**Comment by N-Wouda** [comment]
@lukko @JanVeen would any one of you care to take this one on? This should be a relatively small task, as most of it involves thinking about how much these changes should cost, how it should relate to the current cost of products/initial amount of money, how many things you can have at once (can I extend my warehouse indefinitely?) etc.

**Comment by N-Wouda** [comment]
@LCfP/serious-logistics-games-for-kids anyone? 👍

**Comment by lisavancasteren** [comment]
I managed to find costs for the warehouse options, but for the other factors: all depends quite a lot on the demand. If we can discuss that soon and find a suitable demand for the products, it will be possible to determine the other factors and find realistic trade-offs.

Per factor (product cost, initial amount of money, warehouse changes, truck size/frequency changes), I formulated important points and suggestions for what should be good for a kids-warehouse-game.

[Issue #45 - Investigate the costs for changing warehouse size and size, frequency of incoming trucks.docx](https://github.com/LCfP/Serious-Game-for-Kids/files/640594/Issue.45.-.Investigate.the.costs.for.changing.warehouse.size.and.size.frequency.of.incoming.trucks.docx)
